layouts/shortcodes/img.html

```diff
@@ -1,44 +1,51 @@
-{{ $source := ($.Page.Resources.ByType "image").GetMatch (printf "%s" (.Get "name")) }}
-{{ $customAlt := .Get "alt" }}
-{{ $customSize := .Get "size" }}
-{{ $lazyLoad := default (default true $.Site.Params.GeekdocImageLazyLoading) (.Get "lazy") }}
+{{- $source := ($.Page.Resources.ByType "image").GetMatch (printf "%s" (.Get "name")) }}
+{{- $customAlt := .Get "alt" }}
+{{- $customSize := .Get "size" | lower }}
+{{- $lazyLoad := default (default true $.Site.Params.GeekblogImageLazyLoading) (.Get "lazy") }}
```

Comment:
shouldn't this be Geekdoc

Reply:
I was thinking about this in general and thought it MIGHT be worth switching all the namespaced vars to geeklab and only having a select few which are geeklabGB geeklabGD ... might make things easier? or not.... idk really.

Reply:
Thanks for the hint! I've created a PR to fix it.